### DISCUSSION TITLE
Welcome to project_web3.0 Discussions!

### DISCUSSION THREAD

**Discussion by adrianhajdin** [adrianhajdin/project_web3.0#114] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aScHEABL** [comment]
I added my solution to the problems that people will run into if followed exactly from your youtube video, and you just deleted the entire issue section, now that my solutions are all gone, what are you gonna do with that? I don't even have a copy of it.

My solution was like a paragraph with hundreds of words. wtf man?
![image](https://github.com/adrianhajdin/project_web3.0/assets/22256864/a0684a99-5d8a-431f-ae88-c2646d410b20)

**Reply by monaldmuck** [reply]
can you help with the src >components>navbar
even when copying raw file, my navbar buttons are blacked out and the login button has none of the properties assigned to it

**Reply by aScHEABL** [reply]
@monaldmuck You didn't even share the code. What am I going to do with nothing? Try to ask a good question next time.

**Comment by schow9527** [comment]
I came across this problem "transactionContract.addToBlockchain is not a function at sendTransaction (TransactionContext.jsx:110:61)",but I check all the files, it shows no problems. Im so confused.
